### PR TITLE
CI/CD github workflows

### LINKED ISSUE #22: CI/CD
The CI of Human App backend should run its tests and fail in case of test errors. The implementation will be based on Job Launcher CI.

A reference implementation of CI https://github.com/humanprotocol/human-protocol/blob/2662e1af9b9d6c469845e72aab850a697b8f62b6/.github/workflows/ci-test-job-launcher.yaml

Image

### PR DESCRIPTION
**Pull request by MWBlocky**
## Description

This Pull Request adds the CD/CD workflows

## Summary of changes

- Added docker compose and dockerfile
- Added yaml's for CD/CD
- Updated main package.json

## Related issues
Fixes #22

### CODE REVIEW

**Review by portuu3 — COMMENTED** [review]
are those E2E tests? If the answer is no, why it need REPUTATION_ORACLE_URL?

File: packages/apps/human-app/server/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - '${LOCAL_PORT}:${PORT}'
+    environment:
+      NODE_ENV: ${NODE_ENV}
+      HOST: ${HOST}
+      PORT: ${PORT}
+      REPUTATION_ORACLE_URL: ${REPUTATION_ORACLE_URL}
+      REDIS_PORT: ${REDIS_PORT}
+      REDIS_HOST: ${REDIS_HOST}
+      CACHE_TTL_ORACLE_DISCOVERY: ${CACHE_TTL_ORACLE_DISCOVERY}
+      E2E_TESTING_EMAIL_ADDRESS: ${E2E_TESTING_EMAIL_ADDRESS}
```
[the remark this answers is not in the document]
Comment (MWBlocky) [reply packages/apps/human-app/server/docker-compose.yml]: Fixed in 15194c69

File: .github/workflows/cd-human-app-server.yaml
[the remark this answers is not in the document]
Comment (MWBlocky) [reply .github/workflows/cd-human-app-server.yaml:1]: Fixed in 15194c69

**Comment by MWBlocky** [comment]
> REPUTATION_ORACLE_URL

we use this field in production (user registration and login)

**Review by portuu3 — APPROVED** [review]

**Pull request closed without merging**